docs/services/OneDrive.md

```diff
@@ -289,7 +289,15 @@ Using stream = TryCast((Await file.StorageItemPlatformService.GetThumbnailAsync(
     Await OneDriveSampleHelpers.DisplayThumbnail(stream, "thumbnail")
 End Using
 ```
-  
+
+### Known issues and limitations
+
+Creating a File using the _OpenIfExists_ CollisionOption is not supported for the time being and returns an argument Exception. 

```

Comment:
```suggestion
> [!IMPORTANT]
> Creating a File using the _OpenIfExists_ CollisionOption is not supported for the time being and returns an argument Exception. 
```

We should call this out with the important block to make it more visible when scrolling